spec/models/item_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe Item do 
+  describe "relationships" do 
+    it { should belong_to :merchant }
+    it { should have_many :invoice_items }
+    it { should have_many(:invoices).through(:invoice_items) }
+  end
+
+  describe "validations" do 
+    it { should validate_presence_of :name }
+    it { should validate_presence_of :description }
+    it { should validate_presence_of :unit_price }
+    it { should validate_presence_of :status }
```

Comment:
Same for here with the enums, something like

it { should define_enum_for(:status).with_values([disabled: 0, enabled: 1]) }

Reply:
completed, no `[]`